packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -756,6 +745,23 @@ extension FlutterVmService on vm_service.VmService {
     }
   }
 
+  /// Invoke a flutter extension method, if the flutter extension is not
+  /// available, returns null.
+  Future<Map<String, dynamic>> invokeFlutterExtensionRpcRaw(
+    String method, {
+    @required String isolateId,
+    Map<String, dynamic> args,
+  }) async {
+    final vm_service.Response response = await _checkedCallServiceExtension(
+      method,
+      args: <String, Object>{
+        'isolateId': isolateId,
+        ...?args,
+      },
+    );
+    return response.json;
```

Comment:
I believe this has to be `response?.json` to honor the contract described in the method documentation.